src/run.ts

```diff
@@ -30,12 +19,6 @@ export async function run(context: Context) {
         await onLabelChangeSetPricing(context);
       }
       break;
-    case "label.edited":
-      await watchLabelChange(context);
-      break;
-    case "issue_comment.created":
-      await handleComment(context);
-      break;
     case "push":
```

Comment:
If somebody pushes a ton of commits in one shot, does this run for each? 

This is a reference to our other discussion about how disqualifier runs multiple times

Reply:
No I don't think so, push contains a list of commits if I recall properly. This only happens once when you push only.